exercises/binary-search/binary_search_test.py

```diff
@@ -29,20 +29,24 @@ def test_finds_value_in_array_of_even_length(self):
             5)
 
     def test_identifies_value_missing(self):
-        with self.assertRaisesWithMessage(ValueError):
+        with self.assertRaisesWithMessage(ValueError) as error:
```

Comment:
`self.assertRaisesWithMessage` already checks for `length != 0`

Reply:
Oh, sorry i didn't notice that. I will change the PR.